src/components/SwapPage.tsx

```diff
@@ -107,6 +132,57 @@ const SwapPage = (props: Props): ReactElement => {
     (slippageSelected === Slippages.Custom &&
       parseFloat(slippageCustom?.valueRaw || "0") < 0.5)
 
+  const renderTokenListsWarning = (
```

Comment:
Any reason to have this defined within the component instead of outside?

Reply:
wrapped it in usecallback to avoid rerenders